snuba/cli/jobs.py

```diff
@@ -0,0 +1,32 @@
+from typing import Tuple
+
+import click
+
+from snuba.manual_jobs import Job
+
+
+@click.group()
+def jobs() -> None:
+    pass
+
+
+@jobs.command()
+@click.argument("job_name")
+@click.option(
+    "--dry_run",
+    default=True,
+)
+@click.argument("pairs", nargs=-1)
+def run(*, job_name: str, dry_run: bool, pairs: Tuple[str, ...]) -> None:
+    job_type_class = Job.class_from_name(job_name)
+    if job_type_class is None:
+        raise click.ClickException(
+            f"Job does not exist. Did you make a file {job_name}.py yet?"
+        )
```

Comment:
I think this and line 31 should be combined into a class that lives in snuba/manual_jobs that is something like

`JobLoader.get_job_instance(class_name, **kwargs)`

because that will be useful for both snuba admin and this command. Then, this file should only contain logic related to parsing stuff from the command line